private.tf

```diff
@@ -38,6 +38,17 @@ resource "aws_route_table" "private" {
   }
 
   tags = "${module.private_label.tags}"
+
+  lifecycle {
+    ignore_changes = ["route"]
+  }
+}
+
+resource "aws_route" "private" {
+  count                  = "${length(compact(values(var.additional_private_routes)))}"
+  route_table_id         = "${element(aws_route_table.private.*.id, count.index)}"
+  destination_cidr_block = "${length(compact(values(var.additional_private_routes))) > 0 ? lookup(var.additional_private_routes, element(coalescelist(keys(var.additional_private_routes), list("workaround")), count.index), "0.0.0.0/0") : ""}"
```

Comment:
It won't work with map like `{igw-0038f979 = "10.0.0.2/24"}` because we can have several routes pointing to a single Gateway.
In this case, we will have several equal keys, which is wrong.

It is necessary to use map the following map approach: `{ "10.0.0.2/24" = igw-0038f979 }`.